CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.0.0-RC1
+
+- Added Amazon support
+- Renamed `PurchaserInfo` to `CustomerInfo`, removed some deprecated functions, and deprecated, checkout
```

Comment:
```suggestion
- Renamed `PurchaserInfo` to `CustomerInfo` and removed some deprecated functions. Checkout
```

Reply:
Oh I committed it but I see you made the change already in the other PR